app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="flex flex-col items-center w-full mt-12">
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+    <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name", class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" %>
+  </div>
+
+  
+  <div class="field">
+  <%= f.label :posts_counter %><br />
+  <%= f.number_field :posts_counter, min: 0 , class: "shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline"%>
```

Comment:
- [ ] A new user should not be able to enter the number of posts when signing up because the user at this point has not created a post since the user has no ability to. The posts should only be counted when the user has made a post. Please remove this input field.
![image](https://github.com/peterdtitan/rails-blog/assets/45460192/6adea791-73b4-4f02-b483-2e1c02b91ad1)

Reply:
Adjusted the form fields appropriately.